bundle/compliance/policy/aws_ec2/ensure_public_ingress.rego

```diff
@@ -0,0 +1,33 @@
+package compliance.policy.aws_ec2.ensure_public_ingress
+
+import data.compliance.lib.common
+import data.compliance.policy.aws_ec2.data_adapter
+import future.keywords.every
+
+default rule_evaluation = false
+
+# Ensure that the number of previous passwords that IAM users are prevented from reusing is 24.
+finding = result {
+	# filter 
+	data_adapter.is_nacl_policy
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(rule_evaluation),
+		data_adapter.nacl_entries,
+	)
+}
+
+rule_evaluation {
+	# filter all the entries that 
+	# 1. have ingres (egress == false)
+	# 2. allow any source ip of 0.0.0.0/0
+	entries := [entry | entry := data_adapter.nacl_entries[_]; entry.Egress == false; entry.CidrBlock == "0.0.0.0/0"; entry.RuleAction == "allow"]
+
+	# # make sure that all the matched entries on the filter does not expose any port that is less than 1024
+	# # such as SSH (22) or RDP (3389)
+	every entry in entries {
+		not common.has_key(entry, "PortRange")
+		entry.PortRange.From <= 1024
```

Comment:
maybe I'm missing something, but wouldn't `3389` will pass this check?

Reply:
From CIS 
> A Port value of ALL or a port range such as 0-1024 are inclusive of port 22, 3389, and other remote server administration ports

But you are right that the check is not good enough in that context.
I will add also checks for specific ports

Reply:
I saw it also, but it doesn't make sense? what am I missing?

Reply:
you are right, it does not make any sense.
removing